jupyter_rfb/widget.py

```diff
@@ -113,21 +70,60 @@ class RemoteFrameBuffer(ipywidgets.DOMWidget):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self._ipython_display_ = None  # we use _repr_mimebundle_ instread
         # Setup an output widget, so that any prints and errors in our
         # callbacks are actually shown. We display the output in the cell-output
         # corresponding to the cell that instantiates the widget.
-        self._output_context = OutputContext()
+        self._output_context = RFBOutputContext()
```

Comment:
Is there any reason to make this toggle-able? What if I don't want the output?

I think the error/stdout not being shown by default is to support dashboard-like applications that should only display the widgets involved.

Reply:
I should update that comment, because eventually we decided to *not* do something about print-calls, but provide a `.print()` method instead.

Reply:
Done